src/components/common/Extensions/ColonySwitcher/ColoniesDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { Colony, WatchListItem } from '~types';
+import ColonyItem from './ColonyItem';
+import ColonyAvatar from '~shared/ColonyAvatar';
+import { IColoniesDropdown } from './types';
+import { useSelectedColony } from './hooks';
+
+const displayName = 'common.Extensions.ColonySwitcher.ColoniesDropdown';
+
+const ColoniesDropdown: FC<IColoniesDropdown> = ({ watchlist = [] }) => {
```

Comment:
Why do you use the `I` prefix in the interface name? We use `ComponentProps` convention in this project.